src/libecalc/core/models/compressor/train/chart/variable_speed_compressor_chart.py

```diff
@@ -317,24 +317,25 @@ def _evaluate_point_validity_chart_area_flag_and_adjusted_speed_and_rate(
             # Todo: Need to QA this logic. Below minimum speed requires speed < minimum_speed, but this is not enforced?
             minimum_flow_rate_for_speed = self.minimum_rate_as_function_of_speed(speed_to_use)
             maximum_flow_rate_for_speed = self.maximum_rate_as_function_of_speed(speed_to_use)
-            if rate < minimum_flow_rate_for_speed:
+            if rate == 0:
+                chart_area_flag = ChartAreaFlag.NO_FLOW_RATE
+            elif rate < minimum_flow_rate_for_speed:
                 chart_area_flag = (
                     ChartAreaFlag.BELOW_MINIMUM_SPEED_AND_BELOW_MINIMUM_FLOW_RATE
                     if speed_is_increased
                     else ChartAreaFlag.BELOW_MINIMUM_FLOW_RATE
                 )
-                if increase_rate_left_of_minimum_flow_assuming_asv:
-                    rate_to_use = max(minimum_flow_rate_for_speed, rate_to_use)
-                else:
-                    point_is_valid = False
             elif rate > maximum_flow_rate_for_speed:
                 chart_area_flag = (
                     ChartAreaFlag.BELOW_MINIMUM_SPEED_AND_ABOVE_MAXIMUM_FLOW_RATE
                     if speed_is_increased
                     else ChartAreaFlag.ABOVE_MAXIMUM_FLOW_RATE
                 )
                 point_is_valid = False
-
+            if increase_rate_left_of_minimum_flow_assuming_asv:
+                rate_to_use = max(minimum_flow_rate_for_speed, rate_to_use)
```

Comment:
is `point_is_valid` = `False` here as well?

Reply:
If `increase_rate_left_of_minimum_flow_assuming_asv == True `points will be move to minimum flow and considered to be valid.